setup.cfg

```diff
@@ -0,0 +1,62 @@
+[metadata]
+name = mlfinlab
+version = 0.4.1
+author = Hudson and Thames Quantitative Research
+author_email = research@hudsonthames.org
+licence = BSD-3
+licence-file = LICENSE.txt
+description = Package based on the text book: Advances in Financial Machine Learning, by Marcos Lopez de Prado
+long_description = file: README.md
+long_description_content_type = text/markdown
+platform = any
+url = https://www.hudsonthames.org/
+project_urls =
+    Documentation = https://mlfinlab.readthedocs.io/en/latest/
+    Bug Reports = https://github.com/hudson-and-thames/mlfinlab/issues
+    Project Boards = https://github.com/orgs/hudson-and-thames/projects
+    Source = https://github.com/hudson-and-thames/mlfinlab
+    Blog = https://hudsonthames.org/blog/
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    Intended Audience :: Education
+    Intended Audience :: Science/Research
+    Intended Audience :: Financial and Insurance Industry
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Artificial Intelligence
+    Topic :: Office/Business :: Financial :: Investment
+keywords = 
+    machinelearning
+    finance
+    investment
+    education
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = 
+    ~=3.6
+setup_requires = 
+    setuptools
+install_requires =
```

Comment:
Is there a way to point these requirements to the requirements.txt file? That way we don't need to manually update the values.

Reply:
There are two different requirements. In `install_requires` you should put minimal requirements, only necessary ones. It's well described [here](https://packaging.python.org/discussions/install-requires-vs-requirements/).